website/content/partials/known-issues/1_16-default-lcq-pre-1_9-upgrade.mdx

```diff
@@ -0,0 +1,26 @@
+### Default Lease Count Quota enabled when upgrading from Vault versions before 1.9
```

Comment:
```suggestion
### Default lease count quotas enabled when upgrading from Vault versions before 1.9
```

Style correction: use sentence case for headings, reserve capitalization for product names rather than features